shortbread-website/content/_index.md

```diff
@@ -1,4 +1,9 @@
-# Shortbread Tiles
++++
+title = "Shortbread Tiles"
+toc = false
+breadcrumbs = false
```

Comment:
Can you set this globally so it's not in every file?

Reply:
`breadcrumbs = false` is set everywhere, `toc = false` only on main pages. Will investigate whether this can be set globally. An alternative would be removing breadcrumbs by CSS.

Reply:
Found a way: https://gohugo.io/content-management/front-matter/#cascade